pkg/operator/nfs/controller.go

```diff
@@ -425,6 +449,11 @@ func (c *Controller) onAdd(obj interface{}) {
 	if err := c.createNfsStatefulSet(nfsServer, int32(nfsServer.spec.Replicas)); err != nil {
 		logger.Errorf("Unable to create NFS stateful set %+v", err)
 	}
+
+	logger.Infof("creating nfs server storageclass")
+	if err := c.createStorageClass(nfsServer); err != nil {
+		logger.Errorf("Unable to create NFS stateful set %+v", err)
```

Comment:
change `stateful set` to `storage class`

Reply:
yes will do it.